Utils/Book.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Assignment_1.Utils
+{
+    /// <summary>
+    /// Represents a book in the library system.
+    /// </summary>
+    internal class Book
+    {
+        public int BookId { get; }
+        public string Title { get; }
+        public string Author { get; }
+        public string Genre { get; }
+        public bool IsAvailable { get; set; } = true;
+        public List<User> Issuees { get; } = new List<User>();
+        public DateTime? IssueDate { get; private set; }
+        public string Issuer { get; set; }
```

Comment:
Admin name as the Issuer? Don't you think Id would have made more sense here as there can be more than one admin with same name?